conda_smithy/configure_feedstock.py

```diff
@@ -299,6 +299,12 @@ def _collapse_subpackage_variants(list_of_metas):
     return break_up_top_level_values(top_level_loop_vars, used_key_values), top_level_loop_vars
 
 
+def _yaml_represent_ordereddict(self, data):
```

Comment:
Isn't it a little weird to use `self` as the first argument outside of a method? This implies some binding behaviour that isn't actually there. Maybe `obj` would be better?

Reply:
I agree, that `self` was a little awkward thing I forgot to replace after copy/pasting. Thanks